modules/sdk/src/main/scala/org/tessellation/sdk/domain/snapshot/Validator.scala

```diff
@@ -0,0 +1,34 @@
+package org.tessellation.sdk.domain.snapshot
+
+import cats.syntax.order._
+
+import org.tessellation.ext.cats.syntax.next.catsSyntaxNext
+import org.tessellation.schema.height.SubHeight
+import org.tessellation.schema.snapshot.Snapshot
+import org.tessellation.security.Hashed
+
+object Validator {
+
+  def isNextSnapshot[S <: Snapshot[_, _]](previous: Hashed[S], next: S): Boolean =
```

Comment:
Verify if that method is even used. I remember that it wasn't used at all

Reply:
It was used in snapshot-streaming (but it obviously can't be seen in the tessellation repo) and right now I'm also using it in `SnapshotProcessor`